Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -1023,14 +1023,17 @@ - (void)updatePopupLayout
         [self updateNonWebPopupLayout:statusBarHeight];
         _overlayView.frame = CGRectMake(0, 0, screenWidth, screenHeight);
     }
-    
+   
+    UIEdgeInsets safeAreaInsets = [self safeAreaInsets];
+    CGFloat leftSafeAreaX = safeAreaInsets.left;
     CGFloat dismissButtonX = screenWidth - _dismissButton.frame.size.width - LPMT_ACCEPT_BUTTON_MARGIN / 2;
     CGFloat dismissButtonY = statusBarHeight + LPMT_ACCEPT_BUTTON_MARGIN / 2;
     if (!fullscreen) {
         dismissButtonX = _popupView.frame.origin.x + _popupView.frame.size.width - 3 * _dismissButton.frame.size.width / 4;
         dismissButtonY = _popupView.frame.origin.y - _dismissButton.frame.size.height / 4;
     }
-    _dismissButton.frame = CGRectMake(dismissButtonX, dismissButtonY, _dismissButton.frame.size.width,
+    
+    _dismissButton.frame = CGRectMake(dismissButtonX - leftSafeAreaX, dismissButtonY, _dismissButton.frame.size.width,
```

Comment:
why don't we just move the `- leftSafeAreaX` directly to where `dismissButtonX` is set in line 1029?

Reply:
@gg4race  because if you look at the code the dismissButtonX is assigned 2 times once at line 1029 and another time if(!fullscreen) line 1032.